src/include/daos_task.h

```diff
@@ -304,6 +305,12 @@ typedef struct {
 	struct daos_acl		*acl;
 } daos_cont_update_acl_t;
 
+typedef struct {
```

Comment:
(style)  do not add new typedefs

Reply:
Ignoring for consistency with this file's code style